projects/core/src/checkout/store/effects/checkout.effect.ts

```diff
@@ -53,21 +53,11 @@ export class CheckoutEffects {
               });
             }
             if (payload.userId === OCC_USER_ID_ANONYMOUS) {
-              return [
-                new CheckoutActions.SetDeliveryAddress({
-                  userId: payload.userId,
-                  cartId: payload.cartId,
-                  address: address,
-                }),
-              ];
+              return [new CheckoutActions.SetDeliveryAddressSuccess(address)];
```

Comment:
This looks like a breaking change ( and there are many breaking changes in the component class too )  Let's not forget to address it in the breaking change handling followup ticket.

Reply:
ok, but have to change it here. In backend, `addDeliveryAddress` not only create a new address but also set this address to cart. So, if we don't change this effect, after `addDeliveryAddress` another call `setDeliveryAddress` will be sent.